.github/workflows/semgrep.yml

```diff
@@ -34,6 +34,6 @@ jobs:
 
       - name: Upload SARIF file for GitHub Advanced Security Dashboard
         if: always()
-        uses: github/codeql-action/upload-sarif@v2
```

Comment:
Remove this. This change did not fix the problem. We need a more precise look at Semgrep.